tests/integration/conftest.py

```diff
@@ -102,7 +108,6 @@ def _get_osname():
 
 
 @pytest.fixture(scope="session", autouse=True)
-def download_nm_team_plugin():
-    cmdlib.exec_cmd(
-        ("dnf", "install", "-y", "--downloadonly", "NetworkManager-team",)
-    )
+def download_packages():
+    for package in PACKAGE_LIST:
+        cmdlib.exec_cmd(("dnf", "install", "-y", "--downloadonly", package,))
```

Comment:
No need for a `for` loop, dnf can download multiple packages at once, which is faster. Using
```
cmdlib.exec_cmd(("dnf", "install", "-y", "--downloadonly",) + PACKAGE_LIST)
```
should work. Actually since the package list is only used here, you can probably just write the packages here and drop the constant. I do not see the value of the constant here.

Reply:
Done, thanks!